### DISCUSSION TITLE
A more traditional markdown blogging platform using the same tech as lists.sh

### DISCUSSION THREAD

**Discussion by neurosnap** [neurosnap/lists.sh#10] [Ideas]
Greetings!

I'm thinking about creating a separate blogging platform using the same technology for publishing content as https://lists.sh.

The workflow would be identical, but instead of lists, it would be a traditional blogging platform using markdown.

Would you be interested in a platform like that?

@bentsai pinging you for feedback

**Comment by bentsai** [comment]
👋🏻 @neurosnap!

Yes, on the one hand, I think having a blogging platform that supports Markdown is natural. Markdown is so ubiquitous in this space that it's basically the default.

**But**, I think what lists.sh does is quite interesting because it is expressly *not* Markdown-based. That's what attracts me to it, because it does already support blogging without the overhead of Markdown. I appreciate that I don't have to write any YAML frontmatter. I don't have the option to create inline links. It really focuses the experience on writing prose, and I appreciate that.

When I first saw this, it reminded me of https://github.com/gordonbrander/subtext, which is also line-based. I don't mind that it's different because it is simple enough to learn and parse if I need to. (That project has a little "Why not Markdown?" section, btw).

Also, I was reminded of https://smol.pub, which has a similar vibe. And I realize probably your format is Gemtext compatible.

Having said that, probably more people in the world would want the more traditional blogging with Markdown system with the cool ssh-based experience. Personally, I find the not-markdown refreshing :)

**Reply by neurosnap** [reply]
Thanks for your thoughts!  I agree with you.  There are so many blogging platforms that take markdown and render html.  I think I'll continue to focus on lists.sh for awhile and see what happens.

**Comment by ndom91** [comment]
So initially I thought, this is a no-brainer - of course putting markdown support in here makes sense.

But after having read all your thoughts, I agree that that seems more appropriate for its own spin-off product.

I love this idea in general though and would be happy to help on a markdown supporting fork. I'll keep my eye on this thread :wink: :+1:
